src/components/crew/crew-view/crew-view.component.js

```diff
@@ -38,17 +37,6 @@ class CrewViewPage extends Component {
     captainName: "",
   };
 
-  goVesselViewPage = (vessel) => {
```

Comment:
I refactored this part so that it shares the logic with the other components that need this logic,
now it's in `goVesselViewPage` in `get-data.js`

Reply:
👍